core/domain/user_services.py

```diff
@@ -269,12 +269,15 @@ def require_valid_username(cls, username):
         elif not re.match(feconf.ALPHANUMERIC_REGEX, username):
             raise utils.ValidationError(
                 'Usernames can only have alphanumeric characters.')
-        elif ('admin' in username.lower().strip() or
-              'oppia' in username.lower().strip() or
-              feconf.MIGRATION_BOT_USERNAME in username.lower().strip()):
+        else:
             # Admin usernames are reserved for admins. Note that 'admin'
```

Comment:
Replace this comment with "Disallow usernames that contain the system usernames or the strings "admin" or "oppia".

Reply:
Done!